index.bs

```diff
@@ -947,6 +952,12 @@ spec:css-syntax-3;
     1.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
+    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] then
```

Comment:
```suggestion
    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] and
```

Reply:
Hm, so, I was trying to stay consistent with the wording below. I'm not sure either is necessarily idiomatic but then I don't think either is wrong. Thoughts?

Reply:
Roll the dice and pick one, apply to both :)

slight preference for "and" since we're talking about this

Reply:
and it is, then!